features/FEATURE_BLE/ble/generic/GenericGattClient.h

```diff
@@ -31,7 +32,8 @@ namespace generic {
  * It requires a pal::GattClient injected at construction site.
  * @attention: Not part of the public interface of BLE API.
  */
-class GenericGattClient : public GattClient {
+class GenericGattClient : public GattClient,
```

Comment:
`SigningEventMonitor` is a server interface. Maybe we need another interface to indicates that a signed write command has been sent.

Reply:
No it's not. It can inform of both signed writes and received signed writes. There's no need for Security Manager to implement two EventHandlers to do this. Both the client and the server can implement the same SigningEventMonitor. All the have to implements is set_signing_event_handler